src/OrleansServiceBus/Providers/Streams/EventHub/EventHubQueueCache.cs

```diff
@@ -45,16 +48,32 @@ protected EventHubQueueCache(int defaultMaxAddCount, IStreamQueueCheckpointer<st
         /// <returns></returns>
         protected abstract bool TryCalculateCachePressureContribution(StreamSequenceToken token, out double cachePressureContribution);
 
-        public void Dispose()
+        void IDisposable.Dispose()
```

Comment:
I would disadvise implementing IDisposable explicitly... what prompted this change?